EssentialFeed/EssentialFeediOS/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -10,7 +10,7 @@ public protocol FeedViewControllerDelegate {
 }
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView, FeedErrorView {
-	@IBOutlet private(set) public var errorView: ErrorView?
+	private(set) public var errorView: ErrorView = ErrorView(frame: CGRect(x: 0, y: 0, width: 320, height: 20))
```

Comment:
The frame shouldn't be necessary here. Remove it.

The property can also be lazy.

```suggestion
	private(set) public lazy var errorView = ErrorView()
```